tracer/src/Datadog.Trace/AppSec/Rcm/Models/AsmData/Data.cs

```diff
@@ -13,18 +14,9 @@ internal class Data
     /// <summary>
     /// Gets or sets an integer representing a UNIX timestamp. Past this timestamp, the entry is ignored. Without a timestamp a value never expires.
     /// </summary>
+    [JsonProperty("expiration")]
     public ulong? Expiration { get; set; }
 
+    [JsonProperty("value")]
```

Comment:
Just FYI, I think these changes might cause some practical differences 🤔 For example, previously, `value` would _always_ be provided. Now, by default, it will _only_ be provided if it has a value. We may need to validate that this is the correct and expected behaviour. Also, are these changes actually safe to make, i.e. thinking about backwards compatibility?

Reply:
Yes, these changes are required to match the RC model implemented by the WAF. Until now we did deserialize, merge and feed the WAF with the config. Now we are just fetching the configs directly, and it was complaining, hence the change. And no worries about compatibility, because this version will not work with old WAFS